### PR TITLE
Replace deprecated java Docker container with openjdk (#877)

### PR DESCRIPTION
**Pull request by edouardKaiser**
Backport for 1.1.6

### CODE REVIEW

File: src/sbt-test/docker/override-commands/build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ version := "0.1.0"
 maintainer := "Gary Coady <gary@lyranthe.org>"
 
 dockerCommands := Seq(
-  Cmd("FROM", "dockerfile/java:latest"),
+  Cmd("FROM", "dockerfile/openjdk:latest"),
```
Comment (muuki88) [inline src/sbt-test/docker/override-commands/build.sbt]: Ah, sorry. I saw that afterwards. We need to remove the `dockerfile/`. I will do so and apply it to master and 1.1.x

File: src/sphinx/formats/docker.rst [hunk 2]
```diff
@@ -151,7 +151,7 @@ Docker Base Image
 
 .. code-block:: scala
 
-    dockerBaseImage := "dockerfile/java"
+    dockerBaseImage := "dockerfile/openjdk"
```
Comment (muuki88) [inline src/sphinx/formats/docker.rst]: Same here. We need to remove the `dockerfile/`

File: src/sphinx/formats/docker.rst [hunk 3]
```diff
@@ -195,7 +195,7 @@ In your sbt console type
 .. code-block:: bash
 
     > show dockerCommands
-    [info] List(Cmd(FROM,dockerfile/java:latest), Cmd(MAINTAINER,Your Name <y.n@yourcompany.com>), ...)
+    [info] List(Cmd(FROM,dockerfile/openjdk:latest), Cmd(MAINTAINER,Your Name <y.n@yourcompany.com>), ...)
```
Comment (muuki88) [inline src/sphinx/formats/docker.rst]: And here

File: src/sphinx/formats/docker.rst [hunk 4]
```diff
@@ -260,7 +260,7 @@ Now let's start adding some Docker commands.
   import com.typesafe.sbt.packager.docker._
 
   dockerCommands := Seq(
-    Cmd("FROM", "dockerfile/java:latest"),
+    Cmd("FROM", "dockerfile/openjdk:latest"),
```
Comment (muuki88) [inline src/sphinx/formats/docker.rst]: And here

**Comment by muuki88** [comment]
Release as 1.1.6

File: src/sbt-test/docker/override-commands/build.sbt (hunk 1 above)
Comment (edouardKaiser) [inline src/sbt-test/docker/override-commands/build.sbt]: Not sure I'm following, this is just a cherry-pick from master. There shouldn't be anything to apply on master, no?

Comment (muuki88) [inline src/sbt-test/docker/override-commands/build.sbt]: I guess at the time of this commit `dockerfile/openjdk` was still valid. 🍒-picking was the write thing 😊

Comment (edouardKaiser) [inline src/sbt-test/docker/override-commands/build.sbt]: Thanks for the release

**Pull request merged**